test/system/160-volumes.bats

```diff
@@ -558,4 +558,36 @@ EOF
 }
 
 
+@test "podman volume globs" {
+    vol1a=${PODMAN_TMPDIR}/v1_$(random_string)
+    vol1b=${PODMAN_TMPDIR}/v1_$(random_string)
+    vol2=${PODMAN_TMPDIR}/v2_$(random_string)
+    touch $vol1a $vol1b $vol2
+
+    # if volumes source and dest match then pass
+    run_podman run --rm -v ${PODMAN_TMPDIR}/v1\*:ro $IMAGE ls $vol1a $vol1b
+    run_podman 1 run --rm -v ${PODMAN_TMPDIR}/v1\*:ro $IMAGE ls $vol2
+
+    run_podman create --rm -v ${PODMAN_TMPDIR}/v1\*:ro $IMAGE ls $vol1a $vol1b
+    cid=$output
+    run_podman container inspect $output
+    is "$output" ".*$vol1a" "podman images --inspect should include $vol1a"
+    is "$output" ".*$vol1b" "podman images --inspect should include $vol1b"
+
+    run_podman 125 create --rm -v ${PODMAN_TMPDIR}/v3\*:ro $IMAGE echo fail
+    is "$output" "Error: invalid container path \"ro\", must be an absolute path" "podman create should fail since no glob match"
+
+    skip_if_remote "remote does not support CONTAINERS_CONF*"
```

Comment:
Please don't `skip` in the middle of a test: this will produce a log entry that does not reflect the fact that some tests ran and passed. Instead, please try:
```bash
    if ! is_remote; then
        ...indent the rest
```